src/Surfnet/ServiceProviderDashboard/Application/Command/Entity/RequestDeletePublishedEntityCommand.php

```diff
@@ -36,18 +36,42 @@ class RequestDeletePublishedEntityCommand implements Command
      */
     private $applicant;
 
+    /**
+     * @var string
+     */
+    private $issueType;
+
+    /**
+     * @var string
+     */
+    private $issueSummaryTranslationKey;
+
+    /**
+     * @var string
+     */
+    private $issueDescriptionTranslationKey;
+
     /**
      * @param string $manageId
+     * @param Contact $applicant
+     * @param string $issueType
+     * @param string $issueSummaryTranslationKey
+     * @param string $issueDescriptionTranslationKey
      */
-    public function __construct($manageId, Contact $applicatant)
-    {
+    public function __construct(
+        $manageId,
+        Contact $applicant,
+        $issueType,
```

Comment:
I don't think it should not be the responsibility of the command to have these fields but it should be the responsibility of the handler. Because this prevents coupling in the command and the controller.  Therefore I think they could be better set with DI in the Handler and not through the command. 
```
issueType
issueSummaryTranslationKey
issueDescriptionTranslationKey
```

Reply:
Thanks for the feedback. I'm merging this PR and will include your feedback in the next PR that addresses this logic.